znvis/cameras/trajectories/circular_trajectory.py

```
@@ -0,0 +1,190 @@
+            total_ping_pong_frames = 2 * self.frames_per_rotation
+            if not frame_index // total_ping_pong_frames >= 1:
+                frame_index = frame_index % total_ping_pong_frames
+                if frame_index >= self.frames_per_rotation:
+                    return total_ping_pong_frames - frame_index
+                else:
+                    return frame_index
+            else:
+                frame_index = 0
+                return frame_index
+
+    def get_center_eye_up(self, frame_index: int = None) -> tuple:
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)